go-controller/pkg/ovn/master.go

```diff
@@ -392,6 +392,10 @@ func (oc *DefaultNetworkController) syncNodes(kNodes []interface{}) error {
 		}
 	}
 
+	// Find ACLs with legacy DBIDs and update them from secondary -> user-defined
```

Comment:
we usually run db syncers from here https://github.com/ovn-kubernetes/ovn-kubernetes/blob/master/go-controller/pkg/ovn/default_network_controller.go#L308-L318